### PR TITLE
Document edition field in manifest

### PR DESCRIPTION
**Pull request by maciektr**

### CODE REVIEW

File: website/docs/reference/manifest.md [hunk 1]
```diff
@@ -41,6 +41,20 @@ Scarb bakes in the concept of [Semantic Versioning](https://semver.org/), so mak
    impls or anything else.
 4. Use version numbers with three numeric parts such as 1.0.0 rather than 1.0.
 
+### `edition`
+
+The edition key is an optional key that affects which Cairo edition your package is compiled with.
+The editions allow newer cairo compiler versions to introduce opt-in features that make break existing code.
```
Comment (szymmis) [inline website/docs/reference/manifest.md:47]:
```suggestion
The editions allow newer cairo compiler versions to introduce opt-in features that may break existing code.
```
Is that what was intended, or do I understand it incorrectly?

File: website/docs/reference/manifest.md [hunk 2]
```diff
@@ -41,6 +41,20 @@ Scarb bakes in the concept of [Semantic Versioning](https://semver.org/), so mak
    impls or anything else.
 4. Use version numbers with three numeric parts such as 1.0.0 rather than 1.0.
 
+### `edition`
+
+The edition key is an optional key that affects which Cairo edition your package is compiled with.
+The editions allow newer cairo compiler versions to introduce opt-in features that may break existing code.
```
Comment (szymmis) [inline website/docs/reference/manifest.md:47]: This nitpicking is crazy but as Marek mentioned on Slack, every `Cairo` mention should be uppercase I guess?

**Review by szymmis — APPROVED** [review]

**Comment by mkaput** [comment]
Don't we have to update any guides?

**Comment by maciektr** [comment]
@mkaput Yes, indeed! Sorry. Please take a look at https://github.com/software-mansion/scarb/pull/887

**Pull request merged**